spec/javascripts/karma.conf.js

```diff
@@ -4,10 +4,10 @@
 module.exports = function(config) {
   config.set({
     // base path, that will be used to resolve files and exclude
-    basePath: '../',
+    basePath: '../../',
 
     // testing framework to use (jasmine/mocha/qunit/...)
-    frameworks: ['requirejs', 'mocha', 'chai-jquery', 'chai', 'sinon','sinon-chai', 'jquery-1.11.0'],
+    frameworks: ['requirejs', 'mocha', 'chai-jquery', 'chai', 'sinon','sinon-chai', 'jquery-1.12.4'],
```

Comment:
This should be bumped to 3.3.1

Reply:
This should come through in this PR - https://github.com/moneyadviceservice/frontend/pull/2081/files
which needs a rebase and adding at some point, but it wasn't a priority yesterday

Reply:
👍 Yes, makes sense, let's leave this as is and bump it with that PR.